install/v2plugin/startcontiv.sh

```diff
@@ -26,9 +29,17 @@ fi
 if [ $control_url != ":9999" ]; then
     control_url_cfg="-control-url=$control_url"
 fi
+if [ $vxlan_port != "4789" ]; then
+    vxlan_port_cfg="-vxlan-port=$vxlan_port"
+fi
+
+set -e
 
 echo "Loading OVS" >> $BOOTUP_LOGFILE
-(modprobe openvswitch) || (echo "Load ovs FAILED!!! " >> $BOOTUP_LOGFILE && while true; do sleep 1; done)
+(modprobe openvswitch) || (echo "Load ovs FAILED!!! " >> $BOOTUP_LOGFILE)
+
+#echo "  Deleting vlan,vxlan bridge" >> $BOOTUP_LOGFILE
```

Comment:
If these commented lines are unused, we should delete them.  They can be `git revert`ed or otherwise resurrected later if necessary